deploy/template.yaml

```diff
@@ -419,6 +439,26 @@ Resources:
                 ArnLike:
                   "kms:EncryptionContext:aws:logs:arn": !Sub "arn:aws:logs:${AWS::Region}:${AWS::AccountId}:*"
 
+  CoreFrontSessionsTable:
+    Type: AWS::DynamoDB::Table
+    Properties:
+      # checkov:skip=CKV_AWS_28: Point in time recovery is not necessary for this table.
+      TableName: !Sub "core-front-sessions-${Environment}"
+      BillingMode: "PAY_PER_REQUEST"
+      AttributeDefinitions:
+        - AttributeName: "id"
+          AttributeType: "S"
+      KeySchema:
+        - AttributeName: "id"
+          KeyType: "HASH"
+      TimeToLiveSpecification:
+        AttributeName: "expires"
```

Comment:
Could this be changed to `ttl` to be consistent with how we configure our other Dynamo tables?

Reply:
I'm not sure. I've used expires here as this is by default what the dynamodb-sessions package we are using on the front-end calls the param. We'd need to look and see if this can be configured on the front end to match

Reply:
Ah ok, if it's not an arbitrary choice don't worry, no need to make things more complicated.